lib/wallet/api/http/Cardano/Wallet/Launch/Cluster.hs

```diff
@@ -915,7 +915,7 @@ generateGenesis dir systemStart initialFunds addPoolsToGenesis = do
             -- There are a few smaller features/fixes which are enabled based on
             -- the protocol version rather than just the era, so we need to
             -- set it to a realisitic value.
-            , _protocolVersion = Ledger.ProtVer 7 0
+            , _protocolVersion = Ledger.ProtVer 8 0
```

Comment:
Will enable https://github.com/input-output-hk/plutus/blame/d5f6f4a5505094490f61c8d164515adf7c8f46cc/plutus-ledger-api/src/Plutus/ApiCommon.hs#L147, though not that we have anything in our integration tests that would use it...